exercises/grep/canonical-data.json

```diff
@@ -204,6 +204,37 @@
             "files": ["iliad.txt"]
           },
           "expected": []
+        },
+        {
+          "description": "One file, one match, line number and file flags",
+          "property": "grep",
+          "input": {
+            "pattern": "ten",
+            "flags": ["-n", "-l"],
+            "files": ["iliad.txt"]
+          },
+          "expected": [
+            "2:His wrath pernicious, who ten thousand woes"
```

Comment:
is there a desire to match the behavior of `grep`? If so, it appears that this output should be `iliad.txt`, as I find that `-l` takes precedence over `-n` in following experiments:

```
$ grep -n -l hello <(echo hello)                                                              (10-08 07:58)
/proc/self/fd/11
$ grep -l -n hello <(echo hello)                                                              (10-08 07:58)
/proc/self/fd/11
```

Reply:
Yes you are right. The issue describes the opposite behavior and I didn't check. I will update the test cases accordingly.